src/components/DataDrawer/Chart/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { Bar } from 'react-chartjs-2';
+import colormap from 'colormap';
+import { shuffle } from 'lodash';
+import { ChartConfig } from '../../../config/types';
+import { TableData } from '../../../context/tableStateSlice';
+
+type ChartProps = {
+  title: string;
+  data: TableData;
+  config: ChartConfig;
+};
+
+export function Chart({ title, data, config }: ChartProps) {
+  const header = data.rows[0];
+  const tableRows = data.rows.slice(1, data.rows.length);
+  const colors = shuffle(
+    colormap({
+      colormap: 'rainbow-soft',
+      nshades: tableRows.length,
+      format: 'hex',
+      alpha: 0.7,
+    }),
+  );
+  try {
+    const indices = Object.keys(header).filter(key =>
+      key.includes(config.xAxis || ''),
```

Comment:
No it is a bit more complex than data because of how the data is structure.
Here I am basically getting the keys that are used to hold the data: [Year1, Year2, Year3], that I then use to get labels [2008, 2010, 2012] and the data in each row.

Reply:
We should definitely make it cleaner and a bit more flexible too. Although considering how tricky it gets to setuo a graph, I am more envisioning that adding a graph will be a mix of code and config anyways.

Maybe define the data handling for each graph and expose it as a type for the config, tbd...